.github/workflows/build.yaml

```
@@ -309,7 +337,7 @@ jobs:
     - name: Download partition map
       uses: actions/download-artifact@v4
       with:
-        name: partitions.bin
+        name: partitions${{ matrix.partitions_suffix }}.bin
```

actionlint (1.7.10)

[error] got unexpected character '\u00a0' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)